pkg/controllers/workapplier/metrics.go

```diff
@@ -36,105 +39,87 @@ import (
 // TO-DO (chenyu1): evaluate if there is any need for high-cardinality options and/or stateful
 // metric collection for better observability.
 
-// The label values for the work processing request counter metric.
+// Some of the label values for the work/manifest processing request counter metric.
 const (
-	// The values for the apply status label.
-	workApplyStatusApplied = "applied"
-	workApplyStatusFailed  = "failed"
-	workApplyStatusUnknown = "unknown"
-	workApplyStatusSkipped = "skipped"
-
-	// The values for the availability status label.
-	workAvailabilityStatusAvailable   = "available"
-	workAvailabilityStatusUnavailable = "unavailable"
-	workAvailabilityStatusUnknown     = "unknown"
-	workAvailabilityStatusSkipped     = "skipped"
-
-	// The values for the diff reported status label.
-	workDiffReportedStatusReported    = "reported"
-	workDiffReportedStatusNotReported = "failed"
-	workDiffReportedStatusUnknown     = "unknown"
-	workDiffReportedStatusSkipped     = "skipped"
-)
-
-// Some of the label values for the manifest processing request counter metric.
-const (
-	// The values for the manifest apply status label.
-	manifestApplyStatusUnknown = "unknown"
-	manifestApplyStatusSkipped = "skipped"
-
-	// The values for the manifest availability status label.
-	manifestAvailabilityStatusSkipped = "skipped"
-	manifestAvailabilityStatusUnknown = "unknown"
+	workOrManifestStatusSkipped = "skipped"
+	workOrManifestStatusUnknown = "unknown"
 
-	// The values for the manifest diff reported status label.
-	manifestDiffReportedStatusSkipped = "skipped"
-	manifestDiffReportedStatusUnknown = "unknown"
-
-	// The values for the manifest drift detection status label.
-	manifestDriftDetectionStatusFound    = "found"
-	manifestDriftDetectionStatusNotFound = "not_found"
-
-	// The values for the manifest diff detection status label.
-	manifestDiffDetectionStatusFound    = "found"
-	manifestDiffDetectionStatusNotFound = "not_found"
+	manifestDriftOrDiffDetectionStatusFound    = "found"
+	manifestDriftOrDiffDetectionStatusNotFound = "not_found"
```

Comment:
I think we should try to be consistent with the reason. Our reasons don't use "_"
```suggestion
	manifestDriftOrDiffDetectionStatusNotFound = "notfound"
```